sci-libs/miopen/miopen-4.0.0-r1.ebuild

```diff
@@ -25,8 +25,8 @@ RDEPEND="
 	!opencl? (
 		=dev-util/rocm-clang-ocl-${PV}*
 		sci-libs/rocBLAS[tensile] )
-	!static-boost? ( =dev-libs/boost-1.72* )
-	static-boost? ( =dev-libs/boost-1.72*[static-libs] )"
+	!static-boost? ( >=dev-libs/boost-1.72 )
```

Comment:
[MIOpen requires boost-1.72](https://github.com/ROCmSoftwarePlatform/MIOpen/#prerequisites) to perform kernel cache, and I tried other versions which result in build failure. Can you show me your build.log for MIOpen?

Reply:
@littlewu2508 boost devs have a workaround built into boost for the issues it causes, see this patch:

https://github.com/fuag15/rocm/blob/master/sci-libs/miopen/files/miopen-4.2.0-disable-no-inline-boost.patch

defining that removes the offending preprocessor directives that were confusing llvm-roc